launch-game.sh

```diff
@@ -23,6 +23,7 @@ then
 		elif [ "$TITLE" = "Dune 2000" ]; then MODARG='Game.Mod=d2k'
 		elif [ "$TITLE" = "Tiberian Sun" ]; then MODARG='Game.Mod=ts'
 		else MODARG='Game.Mod=ra'
+
```

Comment:
This line looks like it was accidentally staged, or is there a specific reason for this?

Reply:
my fault. Will remove it soon